input/vlct/run_MHD_shock_tube_test.py

```diff
@@ -0,0 +1,125 @@
+#!/bin/python
+
+# runs VLCT MHD rj2a Shock Tube Tests
+# - This should probably be transitioned to testing framework once it's created
+# - This script expects to be called from the root level of the repository
+#   OR at the same level where its defined
+#
+# These tests draw loose inspiration from shock tube tests used by Athena++.
+# Specifically this script:
+#   1.) Checks that the L1-norm of the error for the from Ryu & Jones (1995)
+#       shock tube problem when the problem evolves along all 3 axes.
+#
+# We should probably try small convergence tests like Athena++
+
+import os.path
+import sys
+import shutil
+from functools import partial
+
+import numpy as np
+
+from testing_utils import \
+    prep_cur_dir, standard_analyze, EnzoEWrapper, CalcTableL1Norm
+
+def run_tests(executable):
+
+    temp = 'input/vlct/MHD_shock_tube/method_vlct_{:s}_rj2a_N{:d}.in'
+    call_test = EnzoEWrapper(executable,temp)
+
+    # can't actually run the serial vs parallel test from this script (not sure
+    # how to get around request for password)
```

Comment:
`export CHARM_ARGS=++local` may to the trick as discussed on Slack.

Reply:
That's an old comment I'll strip that out